### PR TITLE
Publish to RailsBytes via github actions

### PR DESCRIPTION
**Pull request by fargelus**
## What is the purpose of this pull request?
Update RailsBytes template through GitHub Actions. Closes https://github.com/palkan/view_component-contrib/issues/10.

## What changes did you make? (overview)
Added new GitHub Actions workflow for publishing template.

## Checklist

- [x] I've added a Changelog entry
- [x] I've updated a documentation

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -43,7 +43,27 @@ task :build_template do
...
+  ERB.new(contents).result(builder.get_binding)
+end
+
+desc "Push installation template to RailsBytes"
+task :publish_template do
+  require "net/http"
+  require "json"
+
+  tmpl = Rake::Task["build_template"].execute.first.call
+  token, account_id = ENV.fetch("RAILS_BYTES_TOKEN"), ENV.fetch("ACCOUNT_ID")
```
Comment (palkan) [inline Rakefile:55]: Let's use `RAILS_BYTES_ACCOUNT_ID` to make it clearer.

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Publish
+
+on:
+  push:
+    branches:
```
Comment (palkan) [inline .github/workflows/publish.yml:5]: Let's run only on a tag push (`v*`) and `workflow_dispatch` (manual). See, for example, https://github.com/anycable/anycable/blob/master/.github/workflows/push-stable.yml

File: Rakefile [hunk 3]
```diff
@@ -43,7 +43,27 @@ task :build_template do
   builder = TemplateBuilder.new(File.join(__dir__, "app/templates/install"))
   contents = File.read(File.join(__dir__, "app/templates/install/template.rb"))
 
-  puts ERB.new(contents).result(builder.get_binding)
+  ERB.new(contents).result(builder.get_binding)
```
Comment (palkan) [inline Rakefile:46]: Why we removed the `put`, btw?

Comment (fargelus) [reply Rakefile:46]: I suggested that we don't need it since we changed a way to publish template.

Comment (palkan) [reply Rakefile:46]: It's still useful to debug the template locally (I usually do smth like `be rake build_template > tmp/template.rb`)

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - v0.**
```
Comment (palkan) [inline .github/workflows/publish.yml:6]: We can perform this action for all versions

```suggestion
      - v*
```

**Review by palkan — APPROVED** [review]
🚀

**Pull request merged**